src/components/Carousel.tsx

```diff
@@ -31,6 +33,8 @@ interface CarouselProps<T> {
   distance?: number
   onScrollStart?: () => void
   onScrollEnd?: (index: number) => void
+  FooterComponent?: ReactNode
+  style?: StyleProp<ViewStyle>
 }
 
 function Carousel<T>({
```

Comment:
Note: A matter of personal pref, but if we want to be consistent and use arrow function, we can do 

```ts
const Carousel = <T,>() => ....
```

Reply:
Fixed!